internal/bundle/fmt.go

```diff
@@ -69,7 +70,9 @@ func DeprecatedV7_ToV8(data []byte) ([]byte, error) {
 	// this step will unfortunately not produce well-formatted YAML at all
 	// because the proto structures don't have the yaml tags (only the
 	// yac-it structures do) ...
-	interim, err := Format(v8)
+	// for the same reason we convert between the 2 types using JSON since both
```

Comment:
can you add a test case  for it so that we see this works properly

Reply:
I extended the existing test to verify the presence of `scoring_system`